actors/builtin/miner/miner_state.go

```diff
@@ -32,7 +32,7 @@ type State struct {
 	PreCommitDeposits abi.TokenAmount // Total funds locked as PreCommitDeposits
 	LockedFunds       abi.TokenAmount // Total rewards and added funds locked in vesting table
 
-	VestingFunds cid.Cid // Array, AMT[ChainEpoch]TokenAmount
+	VestingFunds []VestingFund
```

Comment:
This should be put behind a CID so we don't have to load it whenever we load the miner's state. See the `Deadlines` structure for how we can do that.

Reply:
Also, instead of storing this as a single array, storing two parallel arrays (or an array and a bitfield?) would be more compact:

```go
type VestingFunds struct {
  Epochs *bitfield.BitField
  Funds []abi.TokenAmount
}
```

(not necessary, but something to consider)

Reply:
This is done.